objc/README.md

```diff
@@ -241,6 +241,14 @@ Instance variables should be camel-case with the leading word being lowercase, a
 id varnm;
 ```
 
+## XIB's naming
+
+XIB files allow to give custom name for it's elements. Therefore each of elements that are part of XIB file should have a proper name that reflects it's purpouse.
```

Comment:
A few typos. Proposal for corrected version of this line;

XIB files allows giving custom names for its elements. Therefore each element that is part of the XIB file should have a proper name that reflects its purpose.